### DISCUSSION TITLE
Medical DICOM viewer

### DISCUSSION THREAD

**Discussion by Mikerhinos** [NVlabs/instant-ngp#1259] [Ideas / Feature requests]
Hello,

not sure if it's the case in other countries but in France when you have a scanner/MRI they give you a CD with all the taken images in a DICOM format.
DICOM viewers are paid solutions and often a bit hard to use, and as MRIs are just "slice" images from x,y and z angle, I wonder if Instant-NGP could use it to build a NeRF about the scanned body part (probably not though as I think that it needs 80% of similarities between images).

**Comment by JoeLoginIsAlreadyTaken** [comment]
You should have a look at https://slicer.org
This is a great Open Source software for reading and processing medical data like DICOM from MRI or CT.
It can also create 3D Views and Objects from slices. (Unfortunately, i only have parts of my skull as MRI Images)
There are also some nice tutorials on Youtube.

The funny thing is, that you can use the RGBA-PNG's von Instant NGP and even view them in 3D, but the Coloring is not optimized for this.
I'm still looking for a simple Viewer for the PNG's from Instant NGP.
